### PR TITLE
Add zalrsc to debug vplan

### PR DESCRIPTION
**Pull request by silabs-krdosvik**
Removed zalrsc part of the TODO.

### CODE REVIEW

File: cv32e40s/docs/VerifPlans/Simulation/debug-trace/CV32E40XS_debug.json
Comment (silabs-robin) [inline cv32e40s/docs/VerifPlans/Simulation/debug-trace/CV32E40XS_debug.json:1]: > Add zalrsc to debug vplan 

> Removed zalrsc part of the TODO.

?

File: cv32e40s/docs/VerifPlans/Simulation/debug-trace/CV32E40XS_debug.json [hunk 1]
```diff
@@ -21,9 +21,9 @@
         "Sub Feature": "",
         "Feature Description": "",
         "Verification Goal": "Add EBREAK or/and C.EBREAK to arbitrary code\nVerify that:\nCore switches into debug mode.\nCurrent PC must be saved to DPC\nCause of debug must be saved to DCSR (cause=1)\nPC is updated to value on dm_haltaddr_i input\nCore starts executing debug code\n\nEnsure exception routine is not enterred",
-        "Pass/Fail Criteria": "Assertion Check",
-        "Test Type": "Other",
-        "Coverage Method": "Assertion Coverage",
+        "Pass/Fail Criteria": "Check against ISS",
+        "Test Type": "Constrained-Random",
+        "Coverage Method": "Functional Coverage",
```
Comment (silabs-robin) [inline cv32e40s/docs/VerifPlans/Simulation/debug-trace/CV32E40XS_debug.json:26]: What are these changes?

They look alright, but they don't seem to match the PR description.

Comment (silabs-krdosvik) [reply cv32e40s/docs/VerifPlans/Simulation/debug-trace/CV32E40XS_debug.json:26]: It looks like I have uploaded an outdated version of the debug vplan (I dont even find the zalrsc extention that I added). Will hopefully fix this in a new commit.

**Comment by silabs-krdosvik** [comment]
Must be somehting wring here, new PR: https://github.com/openhwgroup/core-v-verif/pull/2295

**Pull request closed without merging**